app/src/main/java/org/sil/gatherwords/EntryActivity.java

```diff
@@ -251,6 +259,11 @@ public int getCount() {
             // TODO: Is 0 okay?  Add special create-first-word page?
             return wordIDs.size();
         }
+
+        @Override
+        public int getItemPosition(Object obj) {
+            return POSITION_NONE;
```

Comment:
Drop a comment as to the need of this; it would not be obvious for a future developer.

Reply:
Done.